.github/workflows/dockerhub-description.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - DOCKERHUB.md
```

Comment:
It can be useful to update the description on hub.docker.com between releases but at the same time we don't want to push changes related to new features before the release.

Maybe we should publish on releases but add `workflow_dispatch` to let us manually trigger an update when needed?

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch

Reply:
Good idea. I'll try to add both manual and release event.

Reply:
Due to the [github actions docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-a-list-of-events) i think `[workflow_dispatch, release]:` will work and cover both manual and release events.

Reply:
I think the [Create release](https://github.com/yuzutech/kroki/blob/main/.github/workflows/release.yml#L23) step will trigger the `docker-hub-description` job...

Maybe the `branches:` and `paths:` should be removed then?

Reply:
I prefer to trigger the workflow when a tag is pushed. The `release` event has types associated: published, unpublished, created, edited, deleted, prereleased, released.
So, as far as I understand, it can be triggered more than once for a release/tag.

Since `workflow_dispatch` does not take any option, I would rather use the "long form" syntax:

```
on:
  workflow_dispatch:
  push:
    tags:
      - 'v*'
```

Reply:
> And it seems like the create-release action used in the release.yml is archived. Maybe try something like intuit/auto?

I'm not entirely sure why because it was quite useful... and last time I checked alternatives were not as good/simple. So for now, I'm sticking to it 😅